modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectFieldFormBase.tsx

```diff
@@ -225,6 +228,38 @@ export default function ObjectFieldFormBase({
 		});
 	};
 
+	useEffect(() => {
+		if (
+			Liferay.FeatureFlags['LPS-158962'] &&
+			values.relationshipType === 'oneToMany' &&
+			values.relationshipId
+		) {
+			const makeFetch = async () => {
+				const relationshipData = await API.getRelationship<
+					TObjectRelationship
+				>(values.relationshipId!);
+
+				if (relationshipData.id) {
+					setOneToManyRelationship(relationshipData);
+				}
+			};
+
+			makeFetch();
+		}
+	}, [values.relationshipId, values.relationshipType]);
+
+	const getMandatoryToggleState = () => {
+		if (
+			Liferay.FeatureFlags['LPS-158962'] &&
+			oneToManyRelationship &&
+			oneToManyRelationship.deletionType !== 'disassociate'
```

Comment:
Hi @cgmonte, just a small tweak. How about we use the optional chaining operator?

```suggestion
			oneToManyRelationship?.deletionType !== 'disassociate'
```

Reply:
We tested together and your solution is the best approach, indeed! Thanks @cgmonte! The PR looks good to me!